airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -136,7 +137,7 @@ def chunk_date_range(start_date: DateTime, interval=1) -> Iterable[Mapping[str,
     # Each stream_slice contains the beginning and ending timestamp for a 24 hour period
     while start_date <= now:
         end = start_date.add(days=interval)
-        intervals.append({"oldest": start_date.timestamp(), "latest": end.timestamp()})
+        intervals.append({"oldest": str(start_date.timestamp()), "latest": str(end.timestamp())})
```

Comment:
why is float not working? comment above should be updated

Reply:
Changed to pendulum datest